Library/Homebrew/cmd/audit.rb

```diff
@@ -659,6 +659,28 @@ def audit_line(line, lineno)
     end
   end
 
+  def audit_prefix_has_contents
+    ps = formula.opt_prefix
+    a = "#{ps}/bin"
+    b = "#{ps}/lib"
+    c = "#{ps}/libexec"
+    d = "#{ps}/sbin"
+
+    if Dir["#{a}/*"].empty?
+      if Dir["#{b}/*"].empty?
+        if Dir["#{c}/*"].empty?
+          if Dir["#{d}/*"].empty?
+            problem <<-EOS.undent
+              The installation seems to be empty. Please ensure the prefix
+                 is set correctly and expected files are installed.
+                 The prefix configure/make argument may be case-sensitive.
+            EOS
+          end
+        end
+      end
+    end
+  end
```

Comment:
OK, I know what's wrong.
* First of all, my code is incorrect. I should use `unless`. Clearly, I need some coffee. :cry: 
* Second, `Metafiles.copy?` requires the file name not  full path.

So it should be:

```ruby
 def audit_prefix_has_contents
    return unless formula.prefix.directory?

    unless Dir.glob("#{formula.prefix}/**/*").any? do |f|
         if File.directory?(f)
           false
         else
           f = File.basename(f)
           !(Metafiles.copy?(f) || %w[.DS_Store INSTALL_RECEIPT.json].include?(f))
         end
      end
      problem <<-EOS.undent
        The installation seems to be empty. Please ensure the prefix
           is set correctly and expected files are installed.
           The prefix configure/make argument may be case-sensitive.
      EOS
    end
  end
```